### PR TITLE
Fix documentation for foreignKeysEnabled

### LINKED ISSUE #1150: Possibly wrong documentation for foreignKeysEnabled
I noticed the docs for `foreignKeysEnabled` say as follows:
image

I assumed this means that it only enables validity checks - confirming an object with a foreign key refers to something that exists, etc. 

I was very surprised though to see that setting it to `false` also prevent things like `onDelete: .cascade` from working - even if there is nothing wrong with the foreign keys.

More specifically:
1. Have two records with a `belongsTo` relation with a CASCADE on deletion and a foreign key. 
2. Turn on `foreignKeysEnabled` and make sure there are no warnings/issues.
3. Delete a "Parent" record, I correctly see the cascaded delete of the linked records.
4. Turn off `foreignKeysEnabled`
5. Delete a "Parent" record, all linked records are still there

I'm not sure if this is a bug, or simply a documentation issue but I suspect the latter. If it is, I'm happy to submit a PR with updated docs. 

Thank you 🙏

### PR DESCRIPTION
**Pull request by freak4pc**
Fixes #1150

### CODE REVIEW

File: GRDB/Core/Configuration.swift [hunk 1]
```diff
@@ -6,7 +6,8 @@ public struct Configuration {
     
     // MARK: - Misc options
     
-    /// If true, foreign key constraints are checked.
+    /// If true (the default), support for foreign keys is enabled.
+    /// See https://www.sqlite.org/foreignkeys.html for more information.
```
Comment (groue) [inline GRDB/Core/Configuration.swift:10]:
```suggestion
    /// See <https://www.sqlite.org/foreignkeys.html> for more information.
```

(Based on https://github.com/groue/GRDB.swift/pull/1012)

**Review by groue — CHANGES_REQUESTED** [review]
🎉 Thank you Shai :-)

Please pardon the nit-picking

**Comment by groue** [comment]
I think that's a quick win 🙂

**Comment by freak4pc** [comment]
> 🎉 Thank you Shai :-)
> 
> Please pardon the nit-picking

I am pro nit-picking ;)
Thank you!

**Pull request merged**